sideboard/tests/test_configuration.py

```diff
@@ -74,7 +74,7 @@ def config_overrides_set(self, monkeypatch):
     @pytest.fixture
     def plugin_dirs(self):
         module_path = '/fake/sideboard/plugins/test-plugin/test_plugin'
-        root_path = os.path.join(os.getcwd(), 'plugins', 'test-plugin')
+        root_path = os.path.join(config['plugins_dir'], 'test-plugin')
```

Comment:
Taking a second look, this may actually supposed to be `os.getcwd()`. The directory isn't actually supposed to exist, but it's the directory that will be returned by `get_module_and_root_dirs()` when the tests are running (I think).